Cargo.lock

```diff
@@ -4965,6 +5281,15 @@ dependencies = [
  "cc",
 ]
 
+[[package]]
+name = "which"
+version = "3.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "d011071ae14a2f6671d0b74080ae0cd8ebf3a6f8c9589a2cd45f23126fe29724"
+dependencies = [
+ "libc",
+]
+
 [[package]]
 name = "widestring"
 version = "0.4.3"
```

Comment:
I'm a bit worried at the amount of new dependencies. Would it make sense in any way to make it an opt-in QE feature (in the cargo sense)?

Reply:
Would this then add anything else but more clutter to our code? Because this is a feature we're planning on giving our users to play with...

Reply:
+3MB to the binary... Yeah, it's a bit.